svc-bie-kafka/src/main/resources/application-integration-test.yaml

```diff
@@ -6,13 +6,21 @@ spring:
     # See https://github.com/bitnami/containers/blob/main/bitnami/kafka/README.md#accessing-apache-kafka-with-internal-and-external-clients
     bootstrap-servers: "localhost:9094"
     properties:
+      security.protocol: SASL_SSL
+      security.inter.broker.protocol: SASL_SSL
+      sasl:
+        mechanism: PLAIN
+        username: USERNAME
+        password: PASSWORD
```

Comment:
we'll need to add the password to the Vault/kube secrets and inject it into the containers per env